lib/ansible/modules/cloud/openstack/os_router.py

```diff
@@ -65,7 +65,17 @@
      default: None
    interfaces:
      description:
-        - List of subnets to attach to the router internal interface.
+        - List of subnets to attach to the router internal interface. Default
+          gateway  associated with the subnet will be automatically attached
+          with the router's  internal interface.
+          In order to  provide an ip address  different from  the default
+          gateway, parameters are passed  as dictionary with keys as  network
+          name or ID(net), subnet name or ID (subnet) and the IP of
+          port (portip) from the network.
+          User defined  portip is often  required when a multiple router need
+          to be connected to a single subnet for which the default gateway has
+          been already used.
+
```

Comment:
Although not a showstopper, all of the double spaces within this comment (and some text below, too), could stand to be eliminated.